.github/workflows/rebuild.yml

```diff
@@ -45,6 +52,14 @@ jobs:
         else
           echo "::set-output name=changed::false"
         fi
+    - name: Notify failures on Slack
```

Comment:
a notification that we could not check the hash of a new base image implies that somebody updated the base image without taking the rebuild condition into consideration. it is arguably if this should trigger a slack notification or not